ambari_plugin/common-services/DOLPHIN/1.2.1/package/scripts/dolphin_alert_service.py

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
+
+import time
+from resource_management import *
+
+from dolphin_env import dolphin_env
+
+
+class DolphinAlertService(Script):
+    def install(self, env):
+        import params
+        env.set_params(params)
+        self.install_packages(env)
+        Execute(('chmod', '-R', '777', params.dolphin_home), user=params.dolphin_user, sudo=True)
+
+    def configure(self, env):
+        import params
+        params.pika_slave = True
+        env.set_params(params)
+
+        dolphin_env()
+
+    def start(self, env):
+        import params
+        env.set_params(params)
+        self.configure(env)
+        start_cmd = format("sh " + params.dolphin_bin_dir + "/dolphinscheduler-daemon.sh start alert-server")
+        Execute(start_cmd, user=params.dolphin_user)
```

Comment:
Have you tested what happens if the service is already started at this time and then click Start? Do you need to add the not_if parameter.

Reply:
OK! I will